README.md

```diff
@@ -92,7 +92,7 @@ Arguments:
 | Argument                              | Description                                                                                                     |
 | ------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
 | `dnOrSaslMechanism` (string)          | The name (DN) of the directory object that the client wishes to bind as or the SASL mechanism (PLAIN, EXTERNAL) |
-| `[password]` (string)                 | Password for the target bind DN                                                                                 |
+| `[password]` (string)                 | Password for the target bind DN. For SASL this is instead an optional set of encoded SASL credentials.          |
```

Comment:
@jgeurts It is possible you maybe want to rename `password` to `passwordOrCredentials` but that is maybe too verbose.

Another option would be to start using two functions:
```javascript
client.bind(); // simple authentication
client.bindSASL(); // SASL authentication
```

Maybe the APIs would be more clear then? But that would probably be a separate improvement.

Reply:
@jgeurts did you see this?

Reply:
Yeah, I saw it. For now, I'm ok with the documentation change. If you feel that separating out a bindSASL method would help, I'm ok with that, too.